static/app/components/sidebar/sidebarItem.tsx

```diff
@@ -364,23 +386,32 @@ const StyledSidebarItem = styled(Link, {
   position: relative;
   cursor: pointer;
   font-size: 15px;
-  height: ${p => (p.isInFloatingAccordion ? '35px' : '30px')};
+  height: ${p => (p.isInFloatingAccordion ? '35px' : '40px')};
   flex-shrink: 0;
   border-radius: ${p => p.theme.borderRadius};
   transition: none;
-
-  &:before {
-    display: block;
-    content: '';
-    position: absolute;
-    top: 4px;
-    left: calc(-${space(2)} - 1px);
-    bottom: 6px;
-    width: 5px;
-    border-radius: 0 3px 3px 0;
-    background-color: transparent;
-    transition: 0.15s background-color linear;
-  }
+  margin: ${p => (p.hasNewNav ? space(2) : space(1))} 0;
+  ${p => {
+    if (!p.hasNewNav) {
```

Comment:
you can probs do all of this with a css selector, something like `[data-attr-name]="true" > parent`

Then this could all be driven from a single state change property on the navigation.

Reply:
Ah ehh
I would rather not try to introduce a new pattern for a potentially temporary feature flag :/

Reply:
So - how might you suggest doing this for elements that are not all nested at the same level?

Is there any way to select for the top level rather than just the parent?

Reply:
you can just have the `[data-attr-name="true'] <child-selector> {` and that will select the corresponding children from elements that have that data attribute set (ideally the data-attr selector also has a class selector first so you don't have to check every element in the dom)

Reply:
chatted in slack - this is likely more difficult than i was anticipating with css-in-js